backend/municipal-services/ingestion-service/app/api/endpoints/file_ingestion.py

```
@@ -169,16 +169,17 @@ async def upload_boundaries_excel_sheet(
     get_authorized_request_info(request_info)
 
     try:
-        with tempfile.NamedTemporaryFile(delete=False, suffix=".xlsx") as input_temp_file:
-            content = await boundary_file.read()
-            input_temp_file.write(content)
-            boundary_file_path = input_temp_file.name
+        input_temp_file = tempfile.NamedTemporaryFile(delete=False, suffix=".xlsx")
```

Ruff (0.8.2)

Use a context manager for opening files

(SIM115)